### DISCUSSION TITLE
Do you think there should be more entities?

### DISCUSSION THREAD

**Discussion by TeslaCoil9001** [studio-minus/ppg-bugs-and-requests#2054] [Polls]
‍

**Comment by dinevolence** [comment]
More humans don't really make sense in my opinion. Humans are always the same and it would just be reskins, which a lot of mods already add.

**Reply by TeslaCoil9001** [reply]
Well I just wanted to know what people think

**Reply by dinevolence** [reply]
> Well I just wanted to know what people think

Not saying the poll itself is bad, just saying that that one option doesn't make much sense in my opinion.
